projects/demo/src/modules/experimental/button-group/examples/1/index.less

```diff
@@ -0,0 +1,9 @@
+:host {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+}
+
+[tuiButtonGroup] {
+    width: 21.5625rem;
```

Comment:
Why so specific? What if we move it to :host instead so there's no extra selectors in .less file?